src/main/flight/pid.h

```diff
@@ -382,6 +399,21 @@ typedef struct tpaSpeedParams_s {
     float pitchOffset;
 } tpaSpeedParams_t;
 
+typedef struct aerodynamicsProperty_s {
+    aerodynamicsMode_e mode;
+    float planeMass;
+    float wingLoad;
+    float airDensity;
+    float zeroLiftC;
+    float differLiftC;
+    float zeroDragC;
+    float induceDragC;
+    float liftForceC;
+    float stallAngleOfAttack;
+    float angleOfAttack;
+    bool  isStallWarning;
```

Comment:
Reorder it so that parameters are separated from calculated values
```suggestion
    aerodynamicsMode_e mode;
    float stallAngleOfAttack;

    float airDensity;
    float planeMass;
    float wingLoad;
    float liftZeroC;
    float liftSlopeC;
    float dragParasiticC;
    float dragInducedC;

    float liftActualC;
    float angleOfAttack;
    bool  isStallWarning;
```

And stallAngleOfAttack, isStallWarning is for display only - it can be separated or closer to OSD code (similar to TPA, it is only consumer of estimated flight parameters)